.circleci/config.yml

```diff
@@ -0,0 +1,515 @@
+          command: |
+            echo "The value of IGNORE_CLASS_LIST_PATH=${IGNORE_CLASS_LIST_PATH}"
+            mvn --batch-mode -Dcheckstyle.skip test -f pom.xml \
+                    -Ditext.license=output-config/itextkey.xml \
+                    -Dddp.firecloudKeysDir=output-config/fc_keys \
+                    -Dconfig.file=output-config/testing-circleci.conf  \
+                    -Dmaven.repo.local=<< parameters.m2_repo >> \
+                    -Dsurefire.excludesFile=$IGNORE_CLASS_LIST_PATH   # Key line to do tests in parallel
+
+      - store_test_results:
+          path: target/surefire-reports
+
+  store-jar-job:
+    executor:
+      name: build-executor
+    working_directory: *pepper_apis_path
+    steps:
+      - attach_workspace:
+          at: *root_path
+      - setup-shared-env
+      - store-jar:
+          jar_base_name: DataDonationPlatform
+      - store-jar:
+          jar_base_name: Housekeeping
+
+  deploy-workspace-jar-job:
+    executor:
+      name: deploy-executor
+    working_directory: *pepper_apis_path
+    steps:
+      - attach_workspace:
+          at: *root_path
+      - setup-shared-env
+      - deploy-jar:
+          jar_base_name: DataDonationPlatform
+      - deploy-jar:
+          jar_base_name: Housekeeping
+
+  deploy-stored-jar-job:
+    executor:
+      name: deploy-executor
+    working_directory: *pepper_apis_path
+    steps:
+      - checkout:
+          path: *repo_path
+      - setup-shared-env
+      - retrieve-jar:
+          jar_base_name: DataDonationPlatform
+      - deploy-jar:
+          jar_base_name: DataDonationPlatform
+      - retrieve-jar:
+          jar_base_name: Housekeeping
+      - deploy-jar:
+          jar_base_name: Housekeeping    
+
+workflows:
+  version: 2
+
+  build-test-store-deploy-workflow:
+    jobs:
+      - compile-only-job: &compile_and_test_filters
+          filters: 
+            branches:
+              only:
+                - develop
+                - &rc /^rc.*/
+                - &hotfix /^hotfix.*/
+                - study-server-parallel-tests
+      - parallel-tests-job:
+          parallelism: 4
+          <<: *compile_and_test_filters
+          requires:
+            - compile-only-job
+      - deploy-workspace-jar-job:
+          filters:
+            branches:
+              only:
+                - develop
+                - study-server-parallel-tests
+          requires:
+            - parallel-tests-job
+      - store-jar-job:
+          filters:
+            branches:
+              only:
+                - *rc
+                - *hotfix
+                - study-server-parallel-tests
+          requires:
+            - parallel-tests-job
+
+  deploy-workflow:
+    jobs:
+      - deploy-stored-jar-job:
+          filters:
+            branches:
+              only:
+                - test
+                - staging
+                - production
+                # @todo remove when done
+                - test_deleteme1
+
+
```

Comment:
This is the the other workflow, just waiting for the branches to be pushed to deploy.